src/pages/Login/Login.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import LoginPage from "./Login.page";
+import { MemoryRouter } from "react-router-dom";
+import { AuthContext } from "../../providers/Auth/Auth.provider";
+
+describe("<LoginPage />", () => {
+  test("should render login page correctly", () => {
+    const contextValue = {
+      authed: false,
+    };
+    const { getByTitle } = render(
+      <AuthContext.Provider value={contextValue}>
+        <MemoryRouter>
+          <LoginPage />
+        </MemoryRouter>
+      </AuthContext.Provider>
+    );
+    const page = getByTitle("login-page");
+    expect(page).toBeInTheDocument();
+  });
+});
```

Comment:
Here we can test for the case when the user log in by: Rendering the page, filling the email and password fields with the expected info, triggering the log in button. The expected results could be. 
- If fails: The alert for invalid credentials will be visible
- If success: We will be routed to the notes page. Which means that we can test for the notes tittle to be on screen